src/services/roads/index.ts

```diff
@@ -0,0 +1,44 @@
+const _ = require('lodash');
+import { computeDestinationPoint } from 'geolib';
+const query_overpass = require('query-overpass');
+
+function getBBoxCoordinates(lat, lon, radius) {
+    let points = [];
+    points.push(computeDestinationPoint({lat,lon},radius, 225));
+    points.push(computeDestinationPoint({lat,lon},radius, 45));
+    return points;
+}
+
+function removeDuplicates(array) {
+    let newArray = [];
+    array.map((street) => {
```

Comment:
map should map one array into another. In your case you should do array.filter with your condition, and then map value. Then You will not need newArray variable.